util/raclgen/lib.py

```diff
@@ -25,6 +25,22 @@
     'policies': ['g', 'Dict, specifying the policies of all RACL groups']
 }
 
+# Required fields for the RACL mapping hjson
+mapping_required = {
+}
+mapping_optional = {
+    'registers': ['g', 'Dict, specifying the policy for each register'],
+    'windows': ['g', 'Dict, specifying the policy for each window'],
+    'ranges': ['l', 'List, specifying the policy for each range']
```

Comment:
Each entry in this list has three entries (base, size, policy), right? If so, I suggest rephrasing this to clarify it by referencing the fields of `range_required`.